defusedxml/ElementTree.py

```diff
@@ -29,7 +29,7 @@
 __origin__ = "xml.etree.ElementTree"
 
 def _get_py3_cls():
-    """Python 3.3 hides the pure Python code but defusedxml requires it.
+    """Python 3.3+ hide the pure Python code but defusedxml requires it.
```

Comment:
oh, yeah. I changed the comment in an earlier, less good version of the patch and didn't think to change it again.